### DISCUSSION TITLE
Sixths: Left (3/3), Right (3/3)

### DISCUSSION THREAD

**Discussion by rotkohlsuppe** [rxhanson/Rectangle#1117] [Ideas]
Hi, I'd like to suggest an option for splitting the screen in half and have each half in thirds.
The 9th option comes very close, because it's the only one, which splits the screen into topX, middleX, bottomX.
```
A1 | A2
B1 | B2
C3 | C3
```
Unfortunately only in 9ths.
I'm forward looking for you feedback, and thanks for writing this app!

**Comment by rxhanson** [comment]
Thanks for the suggestion.

At the moment, I don't have plans to add more options like this into Rectangle, as the app is purposefully slim.

You can configure this is [Rectangle Pro](https://rectangleapp.com/pro) (Paid) with the following custom shortcuts:
![image](https://user-images.githubusercontent.com/13651296/225610233-e8f7b33a-0983-4609-b66a-e109c22343ef.png)

At the moment, it's pretty cumbersome to add these in by hand, but I am planning on making this easier in an upcoming Rectangle Pro release.

**Reply by rotkohlsuppe** [reply]
Thank you for pointing out this option with a screenshot. I looked into the description of Rectangle Pro and couldn't see if it would be possible to build it myself.
